src/App.js

```diff
@@ -18,8 +20,9 @@ class App extends Component {
             <MainNavbar />
             <Route exact path="/" component={MainPage} />
             <Route exact path="/user/:username" component={UserPanel} />
-            <Route exact path="/:slug" component={BlogLovWeb} />
+            <Route exact path="/:addblog/slug" component={BlogLovWeb} />
             <Route exact path="/:slug/addpost" component={AddPost} />
+            <Route exact path="/:addblog" component={AddBlog} />
```

Comment:
Again. ":" char is for 'variable' in url. 'addblog' is static route. Remove ":".